src/canvas/SelectableCanvas.ts

```diff
@@ -1400,13 +1404,17 @@ export class SelectableCanvas<
    * @private
    * @param {Object} object to set as active
    * @param {Event} [e] Event (passed along when firing "object:selected")
-   * @return {Boolean} true if the selection happened
+   * @return {Boolean} true if occurred
```

Comment:
can we change those `occurred` with specific information has before `if a different object has been selected` and similarly for the other methods?